app/views/categories/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<nav class="categories-nav">
+  <ul class="category-nav">
+    <li>
+      <%= link_to '<', payments_path, class: "arrow-left" %>
+    </li>
+    <li>
+      <h1 class="transf">Category Details</h1>
+    </li>
```

Comment:
- [ ] This page shows all your transactions, so I think the heading would be better called "Transactions" or "Payments" instead of "Category Details"

Reply:
Update name